### PR TITLE
Create VideoSharing.js

### PR DESCRIPTION
**Pull request by pooiod**
Share your screen or camera to your projects!

VideoSharing

### CODE REVIEW

**Comment by JeremyGamer13** [comment]
extension doesnt seem to ask if the user wants to screenshare? thats probably bad for privacy
also can you add part of your name in the extension ID pls

File: static/extensions/pooiod7/ScreenSharing.js [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+  const videoElement = document.createElement('video');
+  videoElement.style.display = 'none';
+  document.body.appendChild(videoElement);
+
+  let mediaStream = null;
+
+  class ScreenSharing {
+    getInfo() {
+      return {
+        id: 'screensharing',
```
Comment (NexusKitten) [inline static/extensions/pooiod7/ScreenSharing.js:20]:
```suggestion
        id: 'pooscreensharing',
```

**Review by NexusKitten — COMMENTED** [review]
https://github.com/PenguinMod/PenguinMod-ExtensionsGallery/pull/20#issuecomment-1761748728

As for security issues, I feel like it might be okay to not ask for permission here since there's a pop-up anyways for sharing your screen. Plus, for any security we add, someone could just go into the extension and remove it manually 🗿

File: static/extensions/pooiod7/ScreenSharing.js (hunk 1 above)
Comment (RedMan13) [reply static/extensions/pooiod7/ScreenSharing.js:20]: :trollface:

**Comment by pooiod** [comment]
I'm not sure if I should start a new request, but I updated the extension, and added camera capabilities

**Comment by RedMan13** [comment]
if you could supply the new image pls

**Comment by RedMan13** [comment]
maid
GAYix01acAAq_v1

**Comment by RedMan13** [comment]
the images look identical to em
ima try snd test this on le ohoney

**Comment by RedMan13** [comment]
the acreen sharing block throws an error on mobile it seems
Screenshot_20240429-181022

**Comment by pooiod** [comment]
I forgot to mention, mobile browsers do not support screensharing

**Comment by RedMan13** [comment]
boowomp

**Comment by RedMan13** [comment]
yeah i yhink thats it
agter that this can be merged and added

**Comment by pooiod** [comment]
I fixed screensharing errors, and added a boolean so you can check if you have a bad browser, or a Chad browser.

**Comment by RedMan13** [comment]
ima add this yo the menu later

**Pull request merged**